apps/sertra/pages/server/[id].tsx

```diff
@@ -51,6 +52,10 @@ const ServerDetail: React.FC<{
 }> = ({ listing_data, mods_data }) => {
   return (
     <div className={styles.container}>
+      <HeadWrapper
+        title={`${listing_data.name} - V-Rising Servers | Thunderstore`}
+        description={listing_data.description}
```

Comment:
Just a thought, but are we certain we want to use user inputs (description here, name above) as the meta content? E.g. racist content is obviously bad when shown on our website, but can be extra bad when shown on those embeds on other sites?

Reply:
This was a quick suggest by @MythicManiac , I don't have a good understanding/knowledge what should/could be used here. What do you think @MythicManiac ?

Reply:
That's the purpose of the meta fields, we'll just need to deal with moderating them. The alternative is not having any content-bound info available in any embeds, which is also bad.